package.json

```diff
@@ -3999,6 +3999,446 @@
+														"icon": "debug-breakpoint-log",
+														"type": "command",
+														"command": "language-julia.enable-compiled-mode"
+													},
+													{
+														"key": "M",
+														"name": "Disable compiled mode for the debugger",
+														"icon": "debug-breakpoint-unverified",
+														"type": "command",
+														"command": "language-julia.disable-compiled-mode"
+													},
+													{
+														"key": "R",
+														"name": "Restart kernel",
+														"icon": "debug-restart",
+														"type": "command",
+														"command": "language-julia.restartKernel"
+													},
+													{
+														"key": "S",
+														"name": "Stop kernel",
+														"icon": "debug-stop",
+														"type": "command",
+														"command": "language-julia.stopKernel"
+													},
+												]
+											},
+											{
+												"key": "g",
+												"name": "+Go to",
+												"icon": "go-to-file",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "d",
+														"name": "Go to definition",
+														"icon": "symbol-function",
+														"type": "command",
+														"command": "editor.action.revealDefinition"
+													},
+													{
+														"key": "e",
+														"name": "Go to errors/problems",
+														"icon": "error",
+														"type": "command",
+														"command": "workbench.action.problems.focus"
+													},
+													{
+														"key": "g",
+														"name": "Go to definition",
+														"icon": "symbol-function",
+														"type": "command",
+														"command": "editor.action.revealDefinition"
+													},
+													{
+														"key": "r",
+														"name": "Go to references",
+														"icon": "symbol-reference",
+														"type": "command",
+														"command": "editor.action.goToReferences"
+													},
+													{
+														"key": "s",
+														"name": "Go to symbol in file",
+														"icon": "file",
+														"type": "command",
+														"command": "workbench.action.gotoSymbol"
+													},
+													{
+														"key": "R",
+														"name": "Find references",
+														"icon": "symbol-reference",
+														"type": "command",
+														"command": "references-view.findReferences"
+													},
+													{
+														"key": "S",
+														"name": "Go to symbol in project",
+														"icon": "project",
+														"type": "command",
+														"command": "workbench.action.showAllSymbols"
+													}
+												]
+											},
+											{
+												"key": "m",
+												"name": "+Miscellaneous",
+												"icon": "symbol-property",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "c",
```

Comment:
Just looking at https://develop.spacemacs.org/layers/+lang/julia/README.html. Seems like the equivalent of this in spacemacs is in `SPC m s d`. If we move this binding to `SPC m s d`, then I would suggest we rename +Miscellaneous to +Clear and assign the key of `x` with icon of `trash`?